SWIG/discountcurve.i

```diff
@@ -48,6 +50,8 @@ class InterpolatedDiscountCurve : public YieldTermStructure {
 };
 
 %template(DiscountCurve) InterpolatedDiscountCurve<LogLinear>;
+%template(DiscountCurve) InterpolatedDiscountCurve<MonotonicLogCubic>;
```

Comment:
Each curve needs a different name.

Reply:
Just the template part or both template and shared pointer?

Reply:
Just the name inside `%template()`.  It's the name that will be given to the exported class.